src/utils/getParent.tsx

```diff
@@ -0,0 +1,12 @@
+import { PersonLink } from '../components/PersonLink';
+import { Person } from '../types/Person';
+
+export const getParent = (people: Person[], parentName: string | null) => {
+  const parent = people.find(person => person.name === parentName);
+
+  if (parent) {
+    return <PersonLink person={parent} />;
+  }
+
+  return parentName || '-';
```

Comment:
The best way to reverse this case. 

```suggestion
  if (!parent) {
    return parentName || '-';
  }
  
   return <PersonLink person={parent} />;
  
```
But you can update PersonLink component and work on this case there, for encapsulation